### PR TITLE
Removed Mobile Formats links from the book

### PR DESCRIPTION
**Pull request by nishanil**
## Summary

Mobile formats are not updated. Removing the download links.

### CODE REVIEW

File: docs/architecture/microservices/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ To make it easier to get started, the guide focuses on a reference containerized
 
 ## Action links
 
-- Download this e-book in your format of choice (English version only): | [PDF](https://aka.ms/microservicesebook) | [MOBI](https://dotnet.microsoft.com/download/thank-you/microservices-architecture-ebook-mobi) | [EPUB](https://dotnet.microsoft.com/download/thank-you/microservices-architecture-ebook-epub) |
+- This e-Book is also available in a PDF format (English version only) [Download](https://aka.ms/microservicesebook)
```
Comment (mairaw) [inline docs/architecture/microservices/index.md]:
```suggestion
- This e-book is also available in a PDF format (English version only) [Download](https://aka.ms/microservicesebook)
```

File: docs/architecture/microservices/index.md [hunk 2]
```diff
@@ -72,9 +72,7 @@ The purpose of the application is to showcase architectural patterns. **IT IS NO
 
 ## Send us your feedback!
 
-We wrote this guide to help you understand the architecture of containerized applications and microservices in .NET. The guide and related reference application will be evolving, so we welcome your feedback! If you have comments about how this guide can be improved, please send them to:
-
-[dotnet-architecture-ebooks-feedback@service.microsoft.com](mailto:dotnet-architecture-ebooks-feedback@service.microsoft.com)
+We wrote this guide to help you understand the architecture of containerized applications and microservices in .NET. The guide and related reference application will be evolving, so we welcome your feedback! If you have comments about how this guide can be improved, please submit it at https://aka.ms/ebookfeedback
```
Comment (mairaw) [inline docs/architecture/microservices/index.md]:
```suggestion
We wrote this guide to help you understand the architecture of containerized applications and microservices in .NET. The guide and related reference application will be evolving, so we welcome your feedback! If you have comments about how this guide can be improved, submit feedback at <https://aka.ms/ebookfeedback>.
```

**Review by mairaw — APPROVED** [review]
Looks good but left a few suggestions for you to consider

**Comment by nishanil** [comment]
As always, that was very fast @mairaw. Done.

**Pull request merged**